app/views/allocations/edit.html.erb

```diff
@@ -12,4 +12,4 @@
 
 <hr class="govuk-section-break govuk-section-break--l govuk-section-break--invisible"/>
 
-<%= render 'allocations/pom_tables' %>
+<%= render 'allocations/pom_tables_reallocate' %>
```

Comment:
When we are 'editing an allocation' we know that we are actually dealing with a reallocation